anago

```diff
@@ -969,7 +969,7 @@ announce () {
     # so values > 0 are actually truthy
     ((FLAGS_nomock)) && nomock_flag="--nomock"
 
-    logecho "$ release-notify ${nomock_flag} ${RELEASE_VERSION_PRIME}"
+    logecho "$ krel announce ${nomock_flag} -t ${RELEASE_VERSION_PRIME}"
```

Comment:
```suggestion
    logecho "$ krel announce ${nomock_flag} --tag ${RELEASE_VERSION_PRIME}" --print-only
```

Prefer the long-term flag name to make this more readable.

Reply:
Fixed :+1: